digits/model/images/classification/test_views.py

```diff
@@ -70,6 +70,10 @@ class BaseViewsTest(digits.test_views.BaseViewsTest):
 """
 
     @classmethod
+    def setUpClass(cls):
+        super(BaseViewsTest, cls).setUpClass()
```

Comment:
Isn't this done automatically if you don't explicitly declare a `setUpClass()` method? Not a big deal - I'm just curious.